src/p2p/net_node.inl

```diff
@@ -2545,7 +2545,7 @@ namespace nodetool
       peerid_type peer_id_l = arg.node_data.peer_id;
       uint32_t port_l = arg.node_data.my_port;
       //try ping to be sure that we can add this peer to peer_list
-      try_ping(arg.node_data, context, [peer_id_l, port_l, context, this]()
+      try_ping(arg.node_data, context, [peer_id_l, port_l, &context, this]()
```

Comment:
```suggestion
      try_ping(arg.node_data, context, [peer_id_l, port_l, context, this]()
```

Causes segfault, fix suggested on IRC

Reply:
This wasn't supposed to be in this PR (its not necessary). Reverting. I skimmed back through the PR to find other cases, and I didn't spot any. I wish the PR was shorter though; I was probably better off with trying to figure out how to do DOMless output for epee.